packages/@markuplint/file-resolver/src/ml-file.ts

```diff
@@ -32,17 +31,23 @@ export class MLFile {
 		return this.#filePath;
 	}
 
-	async isExist() {
```

Comment:
There is no usage of this metho in the whole code base, so I remove it here, if you think it's unexpect, I'll revert it back and add a sync method too here.